spec/models/preserved_object_spec.rb

```diff
@@ -11,7 +11,7 @@
   end
 
   it 'is not valid without a specified current_version' do
-    expect(PreservedObject.new(druid: 'ab123cd45678', preservation_policy: 'keepit', size: 1))
+    expect(PreservedObject.new(druid: 'ab123cd45678', preservation_policy: 'keepit', size: 1)).not_to be_valid
```

Comment:
I just noticed this myself while working on my ticket.  This is another thing that should be a separate commit.  As I am working on a PR right now refactoring this spec a bit, I suggest you leave this off altogether and see if you're comfy with how I do it in my PR.

Reply:
if my home connection will ever upload my changes ...  (See #74)

Reply:
Sounds good.